src/jsifier.js

```diff
@@ -390,6 +394,9 @@ function ${name}(${args}) {
       if (sig && (RELOCATABLE || ASYNCIFY == 2)) {
         contentText += `\n${finalName}.sig = '${sig}';`;
       }
+      if (RELOCATABLE && isStub) {
```

Comment:
Given stubs are only used in dynamic linking, why does this need to check `RELOCATABLE`?

Reply:
The usage of RELOCATABLE and MAIN_MODULE has been fairly interchangable throughout the codebase.   Its very unusual to set just RELOCTABLE.    Perhaps in this code we should only geneate stubs for `MAIN_MODULE` and not `RELOCTABLE` but the existing code does it based on `RELOCATABLE`.

The stubs effectively allow for delayed/late binding of functions.. so you could argue that this is not needed for `RELOCTABLE` .. but thats really a separate issue.

Reply:
Sorry, I mean: can't this line be this?
```suggestion
      if (isStub) {
```
(since if that local is true, it implies `RELOCATABLE` anyhow. and, I guess `MAIN_MODULE`, but again, that's separate)

Reply:
Done